quick_start.adoc

```diff
@@ -1,22 +1,142 @@
 [#getting-started]
 = Getting started
 
-* <<manage-clusters,Manage clusters>>
-* <<manage-applications,Manage applications>>
-* <<manage-security,Manage security>>
-* <<observability-quick, Observe clusters>>
+* <<introduction,Introduction>>
+* <<create-and-manage-clusters,Create clusters>>
+* <<manifestwork-example, ManifestWork example>>
 
 [#introduction]
 == Introduction
 
 Coming: Architecture and intro.
```

Comment:
Seems like a working note.  Should this be deleted now?

Reply:
Brandi put that in, I was going to leave it for her to remove